packages/wallet/src/redux/actions.ts

```diff
@@ -46,6 +46,7 @@ export const metamaskLoadError = () => ({
 export type MetamaskLoadError = ReturnType<typeof metamaskLoadError>;
 
 // Common Transaction Actions
+// TODO: These should be switched from channelId to processId
```

Comment:
Right now `directFunding` is expecting a `channelId` on the the transaction actions so I'm leaving this in place for now.